File: Codechef/Race_400M/Race_400M.cpp
#include <iostream>
using namespace std;
int main()
{
    int T;
    cin >> T;
    while (T--)
    {
        int X, Y, Z;
        cin >> X >> Y >> Z;
        X = 400/X;
        Y = 400/Y;
        Z = 400/Z;
        if (X > Y && X > Z)
        {
            cout << "ALICE" << endl;
        }
        else if (Y > X && Y > Z)
        {
            cout << "BOB" << endl;
        }
        else if (Z > X && Z > Y)
        {
            cout << "CHARLIE" << endl;
        }
    }
    return 0;
}

File: Codechef/Race_400M/Race_400M.py
T = int(input())
while T:
    T -= 1
    X, Y, Z = map(int, input().split())
    X = 400 / X
    Y = 400 / Y
    Z = 400 / Z
    if X > Y and X > Z:
        print("ALICE")
    elif Y > X and Y > Z:
        print("BOB")
    elif Z > X and Z > Y:
        print("CHARLIE")
